lib/transformers/govukDateInput.js

```diff
@@ -30,6 +31,7 @@ module.exports = ({schemaKey, schema, options, data, schemaErrors} = {}) => {
                 {
                     label: 'Month',
                     classes: `govuk-input--width-2`,
+                    id: `${schemaKey}[month]`,
```

Comment:
This is to address [this](https://github.com/CriminalInjuriesCompensationAuthority/q-transformer/blob/master/lib/transformers/govukDateInput.js#L72) issue